### PR TITLE
Rename to strip-html

### PR DESCRIPTION
**Pull request by LostCrew**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,34 +60,43 @@ Array of strings. Default: `['a', 'b', 'br', 'strong', 'i', 'em', 'mark', 'small
...
 ```
 
+It is also possible to provide custom allowed tags directly to the directive tag, using directive modifiers. This allows local override of the option:
+
+```html
+<template>
+  <!-- only allow p and strong tags -->
+  <div v-strip-html.p.strong="myUnstrippedHTML">
+</template>
+```
```
Comment (LostCrew) [inline README.md:95]: Also document this

File: package.json [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
   "description": "A Vue directive which renders sanitised HTML dynamically",
   "main": "dist/main.js",
-  "repository": "git@github.com:ecosia/vue-safe-html.git",
+  "repository": "git@github.com:ecosia/vue-strip-html.git",
   "author": "Ecosia GmbH <info@ecosia.org>",
+  "contributors": [
+    "Emanuele Marchi (https://github.com/LostCrew)",
+    "Jakub Fiala (https://github.com/jakubfiala)",
+    "Markus Waitl (https://github.com/axlwaii)"
+  ],
```
Comment (LostCrew) [inline package.json:12]: @jakubfiala @axlwaii Let me know if you're ok with this

File: webpack.config.js [hunk 3]
```diff
@@ -5,9 +5,9 @@ module.exports = {
   output: {
     filename: 'main.js',
     path: resolve(__dirname, 'dist'),
-    library: 'VueSafeHTML',
+    library: 'VueStripHTML',
     libraryTarget: 'umd',
-    globalObject: `(typeof self !== 'undefined' ? self : this)`,
+    globalObject: '(typeof self !== \'undefined\' ? self : this)',
```
Comment (LostCrew) [inline webpack.config.js:10]: Linter automatically formatted this

**Review by jakubfiala — APPROVED** [review]
looks awesome!

File: README.md (hunk 1 above)
Comment (jakubfiala) [reply README.md:95]: @LostCrew  sooo I actually just saw this and I'm thinking - doesn't this look like the dev wants to strip *the specified tags*? It reads like "Vue, strip HTML p and strong tags".

I wonder if it's just a consequence of the naming, or if we can have a different way of specifying the tags

**Review by axlwaii — APPROVED** [review]

**Pull request merged**